lib/components/Snackbar/snackbar.scss

```diff
@@ -0,0 +1,97 @@
+@import "@quill/spacing.scss";
+@import "@quill/border.scss";
+@import "@quill/breakpoints.scss";
+@import "@quill/motion.scss";
+@import "@quill/elevation.scss";
+
+.snackbar {
+    display: flex;
+    width: 340px;
+    max-width: 560px;
+    min-height: 48px;
```

Comment:
Do we need to use variable here too?

Reply:
i think tokens arent ready for this yet. i didnt find any token with such dimensions.